lib/transaction/index.js

```diff
@@ -1185,11 +1175,6 @@ function createDistributedTracePayload() {
     return new DTPayloadStub()
   }
 
-  // TODO: This should be removable / covered by insertDistributedTraceHeaders
-  // once the Transaction#createDistributedTracePayload API that directly invokes
-  // this createDistributedTracePayload is fully removed in a future Major version.
-  this._calculatePriority()
-
```

Comment:
This method is never called directly, only via the "headers" method, which calls `_calculatePriority()`